### PR TITLE
AB#419985 companies house data download

### PR DESCRIPTION
**Pull request by BrianKnightEviden**
Added companies house import function

### CODE REVIEW

**Review by RamakrishnaGujjula — DISMISSED** [review]

**Review by deorab — APPROVED** [review]

File: src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Clients/AntivirusClientTests.cs [hunk 1]
```diff
@@ -0,0 +1,123 @@
+﻿using System.Net;
+using AutoFixture.AutoMoq;
+using EPR.SubsidiaryBulkUpload.Application.Clients;
+using EPR.SubsidiaryBulkUpload.Application.Models.Antivirus;
+using EPR.SubsidiaryBulkUpload.Application.UnitTests.Support;
+using Microsoft.Extensions.Logging;
+
+namespace EPR.SubsidiaryBulkUpload.Application.UnitTests.Clients;
+
+// Note, these tests were cloned from WebApiGateway.
```
Comment (tsnatos) [inline src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Clients/AntivirusClientTests.cs:10]: Don't think we need this comment

**Review by mikeymonster — DISMISSED** [review]

File: src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Support/HttpMessageHandlerMockExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
+﻿using System.Net;
+using System.Net.Http.Headers;
+using Moq.Protected;
+
+namespace EPR.SubsidiaryBulkUpload.Application.UnitTests.Support;
+
+// Note, these extensiona were cloned from WebApiGateway.
```
Comment (tsnatos) [inline src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Support/HttpMessageHandlerMockExtensions.cs:7]: Don,t need this comment

File: src/EPR.SubsidiaryBulkUpload.Application/Clients/AntivirusClient.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
+﻿using System.Net;
+using EPR.SubsidiaryBulkUpload.Application.Models.Antivirus;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace EPR.SubsidiaryBulkUpload.Application.Clients;
+
+// Note, this code was cloned from WebApiGateway.
```
Comment (tsnatos) [inline src/EPR.SubsidiaryBulkUpload.Application/Clients/AntivirusClient.cs:8]: Don't need this

File: src/EPR.SubsidiaryBulkUpload.Application/Clients/IAntivirusClient.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Net;
+using EPR.SubsidiaryBulkUpload.Application.Models.Antivirus;
+
+namespace EPR.SubsidiaryBulkUpload.Application.Clients;
+
+// Note, this code was cloned from WebApiGateway.
```
Comment (tsnatos) [inline src/EPR.SubsidiaryBulkUpload.Application/Clients/IAntivirusClient.cs:6]: Remove comment

**Review by tsnatos — COMMENTED** [review]
Should remove the unnecessary comments.

File: src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Clients/AntivirusClientTests.cs (hunk 1 above)
Comment (mikeymonster) [reply src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Clients/AntivirusClientTests.cs]: Removed

File: src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Support/HttpMessageHandlerMockExtensions.cs (hunk 2 above)
Comment (mikeymonster) [reply src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Support/HttpMessageHandlerMockExtensions.cs]: Removed

File: src/EPR.SubsidiaryBulkUpload.Application/Clients/IAntivirusClient.cs (hunk 4 above)
Comment (mikeymonster) [reply src/EPR.SubsidiaryBulkUpload.Application/Clients/IAntivirusClient.cs]: Removed

**Review by tsnatos — APPROVED** [review]

**Review by mikeymonster — APPROVED** [review]

File: src/EPR.SubsidiaryBulkUpload.Application/Clients/AntivirusClient.cs (hunk 3 above)
Comment (mikeymonster) [reply src/EPR.SubsidiaryBulkUpload.Application/Clients/AntivirusClient.cs]: Removed

**Pull request merged**